types/recurly__recurly-js/test/paypal.ts

```diff
@@ -9,14 +21,4 @@ export default function paypal() {
   paypal.on('fake-event', () => {});
 
   paypal.start();
-  paypal.start({
```

Comment:
The recurly documentation states that `payPal.start` contains the `options` parameters. Could you please add them back?

Reply:
good call!

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43900/commits/d8aa8b343e40e4a0a7c83058ad242b0d9c8313ab